spec/models/bus_stops_route_spec.rb

```diff
@@ -35,12 +35,12 @@
       end
     end
   end
-  describe 'import' do
+  describe 'establish_sequences' do
```

Comment:
This was changed (also written) in #111. The name change was suggested in the review process, and then the specs didn't get updated.